docs/concepts/hosted-workflow-architecture.md

```diff
@@ -5,7 +5,7 @@
 Whilst it is possible to use the Marain workflow engine in-process by simply consuming the Marain.Abstractions package and the appropriate storage packages, we strongly recommend it is deployed as standalone services using the provided Azure Function apps. Using these functions gives some extremely useful features out of the box.
 
 - Scalability: deploying the functions apps using the consumption-based hosting plan allows scaling out trigger processing across up to 200 instances.
-- High-volume message ingestion: If it is necessary to ingest extremely high volumes of triggers, but there is less need for them to be processed quickly, a separate ingestion endpoint can be used which uses Event Hubs to ingest and queue messages for processing *.
+- High-volume message ingestion: If it is necessary to ingest extremely high volumes of triggers, but there is less need for them to be processed quickly, a separate ingestion endpoint can be used which uses Event Hubs to ingest and queue messages for processing \*.
```

Comment:
You've escaped the asterisk here, but wasn't this a reference to a footnote that also had an asterisk? Doesn't that also need to be escaped? I don't see such a change (although with GitHub's struggles in dealing with long-running PRs it's hard to know).

Reply:
Maybe it was already escaped? I've checked, and it definitely is...